css/styles.css

```diff
@@ -6,10 +6,35 @@ header {
   background-color: #4c2474;
 }
 
+a {
+  padding-left: 3rem;
+  font-size: 1.5rem;
+}
+
+.entries-title {
+  position: relative;
+}
+
+a.new {
+  position: absolute;
+  right: 0;
+  top: 0;
```

Comment:
Works, but I would have probably done this with flexbox rather than positioning. Your columns can also act as flex parents as well